install.sh

```diff
@@ -414,6 +511,16 @@ while true; do
     --cuda|--use-cuda)
         build_cuda=true
         shift ;;
+    --cudapath)
+	cuda_path=${2}
+	CUDA_BIN_PATH=${cuda_path}
```

Comment:
Not all bash variables are inherited by the child processes of the script. Only exported variables are visible to child processes. This defines `CUDA_BIN_PATH` for `install.sh`, but it won't be visible in your `cmake` calls unless you add `export CUDA_BIN_PATH`. Since this variable is not exported and is not referenced anywhere else in the script, it has no effect.

Alternatively, you could move `CUDA_BIN_PATH=${cuda_path}` to be a variable specified just for the CMake invokation, like `CXX=${compiler}` is.